README.md

```diff
@@ -90,15 +92,6 @@ Versions are auto incremented based on commit messages and SemVer (Major.Minor.P
 - `patch:` or `fix:` example `fix: url bug` 
   - this will increment the PATCH version eg 1.0.0 to 1.0.1. This will also happen by default.
 
-#### Build process
-
-1. Pushes to any branch will start the build process
-2. `.circleci/circle_trigger.sh` will check for any changes in our packages, and if builds have failed previously
```

Comment:
Hi! Please change to:

2. `.circleci/circle_trigger.sh` will check for any changes in our packages and whether builds have failed previously.

Reply:
👋 - these are line removals (so ignore them!)